protobufs/content/setup.cfg.in

```diff
@@ -0,0 +1,25 @@
+[metadata]
+name = p4runtime
```

Comment:
Should this be called `p4runtime-dev` or even `ipdk-io-p4runtime-dev` to differentiate from upstream p4runtime?

Reply:
We're currently differentiating our variant by using a distinctly different versioning scheme.

I think we want to use the same name as the existing module, so people don't have to change their programs to import something different.

Which is not to say that injecting `ipdk-io` into a name to distinguish it is a bad idea. I may steal it at the next appropriate opportunity... 😉